### PR TITLE
Adds the Coderbus and Adminbus bar signs: No merge conflicts edition

### PR DESCRIPTION
**Pull request by ghost**
theadminbus            thecoderbus

I saw this suggestion in the PR for re-adding the Grey Tide sign and felt I needed to do it justice.
Also changes the description on all the bar signs (usually just adding periods).

I didn't see the need to add a changelog for something so minor but if you want one lemme know.

This is the second PR since I was having trouble with the first one due to merge conflicts arising from a different change being merged before this one.

P.S. I'm pretty happy with the sprite so I don't think I'm going to change it.

### CODE REVIEW

**Comment by Nienhaus** [comment]
It's actually starting to make me think of an IRC client which works out well.

File: code/game/objects/structures/barsigns.dm [hunk 1]
```diff
@@ -210,12 +210,19 @@
...
 /datum/barsign/thenest
 	name = "The Nest"
 	icon = "thenest"
 	desc = "A good place to retire for a drink after a long night of crime fighting."
 
 
+/datum/barsign/thecoderbus
+	name = "The Coderbus"
+	icon = "thecoderbus"
+	desc = "A very controversial establishment renowned for its constantly changing drinks. It smells faintly of paprika."
```
Comment (Incoming5643) [inline code/game/objects/structures/barsigns.dm]: [requisite pointing out of the obvious without further comment]

Comment (Dorsisdwarf) [inline code/game/objects/structures/barsigns.dm]: It's not a player reference.

Because he'll never play again

**Comment by Niknakflak** [comment]
Still a reference to a once existing player so -1

**Comment by Xhuis** [comment]
:^)

**Comment by ColonicAcid** [comment]
please merge this before the 24 hours
and then post the salt image as a repeated response like the witty jokers that we all are


of course we were only pretending to be retarded lol!!!!

**Comment by RemieRichards** [comment]
technically this PR is already beyond the 24hr limit (it's a remade PR because of conflicts)
so we /can't/ merge before 24 :P

**Comment by Ricotez** [comment]
Add one for the Adminbus too, with a hammer for the logo.

And heck, why not the Conspiracybus too. Make the logo for that one a tinfoil hat.

**Comment by dannno** [comment]
can't you be just a little bit more subtle

**Comment by Xhuis** [comment]
Adminbus is coming up. Not gonna do the conspiracybus though as it doesn't fit.

Also yes, I am going to add a changelog.

**Comment by Xhuis** [comment]
Uuuuugh not again!

**Comment by Xhuis** [comment]
Mmkay. I added the Adminbus sign and fixed all the issues, we're good now

**Comment by dannno** [comment]
the S on the end of adminbus is awkwardly spaced in comparison to the rest of the sign

**Comment by Xhuis** [comment]
i know
i'm shit at spriting

**Comment by Xhuis** [comment]
All right done, GIF is not consistent though

**Comment by Niknakflak** [comment]
That hammer looks terrible.  Dont be afraid to use colors man.

**Comment by dannno** [comment]
agreed

**Comment by Xhuis** [comment]
I can't sprite. I'm terrible at it - that's basically the best I can do :monkey_face:

**Comment by Miauw62** [comment]
@nienhaus @WJohn

**Comment by dannno** [comment]
merge this

**Comment by dannno** [comment]
MERGE THIS

**Comment by Miauw62** [comment]
@nienhaus @WJohn you are the sprite kings ;-;

**Pull request merged**